packages/block-library/src/group/edit.js

```diff
@@ -20,6 +22,10 @@ function GroupEdit( { attributes, className, clientId } ) {
 
 	return (
 		<BlockWrapper className={ className }>
+			<BoxControlVisualizer
+				values={ attributes.style?.spacing?.padding }
+				showValues={ attributes.style?.visualizers?.padding }
+			/>
```

Comment:
I think this can probably be absorbed by the hook and added automatically somehow. cc @ItsJonQ

Reply:
@ItsJonQ I'm not familiar with the attribute shape yet, can you clarify a bit what `visualizers` contain here?

Reply:
It'd be nice if it would be absorbed by the hook, otherwise we need to be more careful with any updates to `BoxControlVisualizer`. Also from its name, it feels like it shouldn't be a public API.

Reply:
@youknowriad Sure! I can expand on this a bit. The visualizers need to show in 2 ways:

1. On value change (they flash) (via the `values` prop)
2. On hover (they persist) (via the `showValues` prop, which are `boolean` values)

Right now, state for `showValues` object is being communicated from the controls (in the sidebar) to the canvas via updating the `style` attribute.

It works, but I'd prefer a more streamlined way of coordinating this state - Ideally without using the data store and relying on a `context` + Provider wrapper.

Hope this helps!

Reply:
Yeah, I'm a bit concerned about the use of an attribute for that state since it's not a persistent state. 
I wonder if we could just rely on local state if we move the rendering of the `BoxControlVisualizer` to the hook. Not sure thought.